pages/Integrations_and_Agents/Agents/Configure-nOps-Kubernetes-Agent-on-EKS.md

```diff
@@ -325,6 +325,10 @@ The following table lists the optional configuration parameters for the KarpenOp
 | `containerInsights.imageTag` | Image tag for the nOps Container Insights Agent container image | `2.0.4` |
 | `containerInsights.successfulJobsHistoryLimit` | Number of successful finished jobs to keep for the nOps Container Insights Agent | `2` |
 | `containerInsights.failedJobsHistoryLimit` | Number of failed finished jobs to keep for the nOps Container Insights Agent | `2` |
+| `containerInsights.resources.limits.cpu` | CronJob CPU Limit | `500m` |
```

Comment:
Instead of CronJob change to Container Insights Agent

Reply:
good call